components/crossplane-config/base/rbac.yaml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: crossplane-sa
+  namespace: crossplane-config
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: crossplane-clusterrole
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - secrets
+  - serviceaccounts
+  verbs:
+  - create
```

Comment:
I believe we need to grant all verbs (`"*"`) for each rule listed here. Crossplane will create but then eventually need to access and delete these resources upon removal of a claim.

Reply:
updated